content/02.main-text.md

```diff
@@ -442,7 +442,7 @@ These guidelines did not discuss broader community norms that may have improved
 It is also important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
 Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1109/hicss.2007.277].
 <!-- cited a previous version of doi:10.1080/08874417.2009.11645338 (see https://git.io/vdSSR) -->
-Conferences and open source projects have used codes of conduct to establish these norms [@url:http://geekfeminism.wikia.com/wiki/Code_of_conduct_evaluations?oldid=35801; @url:https://www.contributor-covenant.org/].
+Conferences and open source projects have used codes of conduct to establish these norms [@doi:10.1109/SANER.2017.7884606].
```

Comment:
https://ieeexplore.ieee.org/document/7884606 includes links to both the geekfeminism wiki and contributor covenant and is a more comprehensive discussion of open source COCs. I was considering keeping the geekfeminism wiki references as a link to "have used", but the wiki hasn't been updated since 2017, so I don't think it's adding anything beyond the current citation.